pkgs/by-name/ta/tangle/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  fetchurl,
+  # Build depends
+  cmake,
+  qt6,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Using `finalAttrs:` is preferred nowadays (when possible), because it makes overriding much easier.